build.gradle

```diff
@@ -160,19 +160,14 @@ dependencies {
 
     compileOnly 'com.oracle.substratevm:svm:19.2.0'
 
-    testCompile group: 'junit', name: 'junit', version: '4.12'
-    
-    def junitVintageVersion = '5.2.0'
     def junitJupiterVersion = '5.5.1'
     def junitJupiterParamVersion = '5.4.2'
```

Comment:
you should use the same version here 5.5.1

Reply:
resolved in #98